webpack.config.prod.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/wrap-multilines */
+const path = require('path');
```

Comment:
#### Nitpick
- Are you performing minimization in this file?

Reply:
@aanchirinah Webpack is doing minification by default. So it isn't done in this config but in the build script in `package.json`

Reply:
I don't see any `UglifyJSPlugin` in your package.json @thislekan , can you explicitly tell me Webpack 4 is doing minification properly without `UglifyJSPlugin` installed?

Reply:
@aanchirinah this is a link to how minification is done by default in production [Webpack 4 minification](https://webpack.js.org/guides/production#minification)

Reply:
@thislekan , okay thanks for the update , I'm still old school webpack 1.